### ISSUE TITLE
JMM Server crash on 3.4.2.16 official release.

### ISSUE THREAD

**Issue by celcius01** [japanesemediamanager/jmmserver#125]
Yep, just like the title said.
I've read the breaking changes; config fixer said that my server config doesn't need changes but desktop config file was successfully fixed, i have the .net framework 4.5.2.
Server crash during the initializing host process.

The usual:
https://www.dropbox.com/sh/v184wuxwecnk3rm/AABJkQXfpxCNRDj6feD9drEPa?dl=0

Thanks Leo

**Comment by jmediamanager** [comment]
Anything in your event viewer for windows?
Were you previously on 3.1.32?

**Comment by celcius01** [comment]
Event viewer? Uhh... i don't know?
Anyway, yes i was on 3.1.32. Before that i was using the beta for a while.
What do i need to check on the event viewer?

**Comment by celcius01** [comment]
Hmm... i've plenty of errors regarding this on the event viewer. I'll put it in the dropbox so just wait a sec.

**Comment by jmediamanager** [comment]
Ok thanks, yes the Windows Event Viewer (application)

If the beta worked this should as well.
Make sure you don't have any other instances of JMM Server running.

You could also attach your JMM Server config file (with usernames/passwords removed)

**Comment by celcius01** [comment]
Here you go:
https://www.dropbox.com/sh/b606sfktpjjingy/AADawQP1FoEP6JdhkYyBX9bza?dl=0

I only put 3 of the errors as the rest are just the same recurring ones when i retried to start the server. Tell me if you need anything else ok?

**Comment by jmediamanager** [comment]
This is a strange error indeed.

It is failing to set up one of the HTTP channels.
Have you restarted your machine since you installed .NET?

**Comment by celcius01** [comment]
.net was already installed before all this. I've also repaired it from your link, but havent restart. I'll try to restart right now & get back to you.

**Comment by celcius01** [comment]
Nope... still crash.

**Comment by jmediamanager** [comment]
Hmm, I still can't replicate this error (and no-one has reported a problem)
Do you have another computer which you can test this on?

No need to set up a database, just try and run JMM Server

**Comment by celcius01** [comment]
Ok, i'll try it on another one and get back to you.

**Comment by celcius01** [comment]
Tried on my old laptop, which still have both jmmd @ jmms official (3.1.32) and it worked perfectly.
This is funny... It should working, right? Because the beta worked fine before.
I tried copying the old config from the other laptop (the 3.1.32) and paste it here, nothing too.
Tried to install it in safe mode, heck i can't even access safe mode. It's all black screen.
Tried to run it with the firewall off, still crash.

**Comment by celcius01** [comment]
Hmm i think i'm gonna stick with 3.1.32 for the time being as it was nice to me all this time so far. 
So don't pull your hair over this too much alright Leo? :D 
Anyway i'm off to the sea for about half a year, so you'll have all the time in the world to try to figure this out, if you want. If not then i'll just continue with this one, no problem here. It's still great without all the fancy stuff so i'm still fine.
Anyway, thanks again Leo. See ya again somewhere end of the year~

**Comment by CatmanIX** [comment]
I'm also crashing on launch, but i don't know what breaking changes or config fixer celcius is talking about?

**Comment by jmediamanager** [comment]
Have u looked at the following?
http://japanesemediamanager.github.io/downloads.html

Breaking changes

- NET Framework 4.5 is now required - Download from Microsoft
- If you are upgrading from 3.3 or lower JMM Server and Desktop both need config changes. JMM Server will crash if you try to run it without fixing first. I have included a 'ConfigFixer' app above

**Comment by CatmanIX** [comment]
Ah, I missed that somehow, bit distracted i guess. ConfigFixer worked, thanks.

**Comment by celcius01** [comment]
Fixed this today somehow. 
I was using a custom theme for windows and to do that i need to patch some of the windows files using UXTheme multi-pacther (i'm using version 13.1).
This patch made me unable to access my Safe Mode, it made it black screen with blinking cursor. Found that out the other day by retracing back all the things in the world i did to this machine.
Anyway then it struck me today that maybe it caused me to unable to run JMMS properly, so:

1. I restore the patched files so that my system files are back to normal.
2. Went to Safe Mode with Networking.
3. Ran JMMS and update from there. (After installing at this point, the database can't be started. I figured that maybe because it's in Safe Mode.)
4. Ran JMMD and update it from there too. (Server to 3.4.2.19, Desktop to 3.4.2.18)
5. Shut down both server & desktop and restarted windows normally.
6. Ran JMMS and it ran like an olympic gold medalist. JMMD too.
7. Then i repatched my windows so that i could run my custom theme again, during this time both the server & desktop were already shut down.
8. After repatching & applying my theme, i restarted the server & desktop. And thank god they ran.

Not sure what happened though. I decided to update because just the other day suddenly anidb starts banning me like there's no tomorrow. I've waited until more than 48 hours even but still the ban was there. Then i saw that misakitchi had the same problem. Luckily this is fixed now.
Closing this happily~
